package.json

```diff
@@ -27,12 +27,15 @@
   "author": "Devhub Team",
   "license": "ISC",
   "devDependencies": {
-    "@commitlint/cli": "^19.0.3",
-    "@commitlint/config-conventional": "^19.0.3",
+    "@commitlint/cli": "^19.3.0",
+    "@commitlint/config-conventional": "^19.2.2",
     "husky": "^9.0.11",
     "lerna": "^8.1.2"
   },
   "workspaces": [
     "./packages/*"
-  ]
+  ],
```

Comment:
**question (blocking):** Please, is this dependency required for any packages on the root level?